typings/codePush.d.ts

```diff
@@ -187,10 +187,11 @@ interface CodePushCordovaPlugin {
      * Calling this function is required on the first run after an update. On every subsequent application run, calling this function is a noop.
      * If using sync API, calling this function is not required since sync calls it internally.
      *
+     * @param deploymentKey Optional string deployment key for this package.
      * @param notifySucceeded Optional callback invoked if the plugin was successfully notified.
      * @param notifyFailed Optional callback invoked in case of an error during notifying the plugin.
      */
-    notifyApplicationReady(notifySucceeded?: SuccessCallback<void>, notifyFailed?: ErrorCallback): void;
+    notifyApplicationReady(deploymentKey?: string, notifySucceeded?: SuccessCallback<void>, notifyFailed?: ErrorCallback): void;
```

Comment:
I don't mind adding deploymentKey to notifyApplicationReady, but it needs to be the last argument. Not only does it break the API if it's at the front, but it's more consistent with checkForUpdate's custom deployment key specification if it's at the back.